wemake_python_styleguide/logic/baseline.py

```diff
@@ -0,0 +1,120 @@
+import json
+import os
+from collections import Counter, defaultdict
+from hashlib import md5
```

Comment:
some distributions compile out `md5` so you'll probably want a stronger hash function

Reply:
Sure! Any suggestions based on your experience?

Reply:
sha256 should do